fs/sync_test.go

```diff
@@ -818,7 +818,7 @@ func testServerSideMove(t *testing.T, r *fstest.Run, withFilter bool) {
 
 	// Do server side move
 	fs.Stats.ResetCounters()
-	err = fs.MoveDir(FremoteMove, r.Fremote)
```

Comment:
I'd like to see a test with the flag set to true.  I thought you did one before but I don't see it!

Reply:
done 👍 
I'd implemented `fstest.CheckRootDir` before, which is now obsolete because we are no longer deleting the root dir.